unity/Assets/Scripts/ImageSynthesis/ImageSynthesis.cs

```diff
@@ -427,7 +431,7 @@ enum ReplacelementModes {
         var renderRT = (!needsRescale) ? finalRT :
             RenderTexture.GetTemporary(mainCamera.pixelWidth, mainCamera.pixelHeight, depth, format, readWrite, antiAliasing);
         if (tex == null) {
-            tex = new Texture2D(width, height, TextureFormat.RGB24, false);
+            tex = new Texture2D(width, height, TextureFormat.RGBA32, false);
```

Comment:
Is this `Encode` method called elsewhere? It could be good to parametrize the textureformat/bit-depth so that only the code calling for depth uses 32-bits/RGBA32

Reply:
The 2D texture is used by all the other passes.  So the segmentation cameras also re-use the same texture. I have updated the Python code to handle receiving a 4 byte frame versus a 3 byte frame with the PR.